packages/Chem/src/widgets/scaffold-tree.ts

```diff
@@ -1512,3 +1544,56 @@ class SketcherDialogWrapper {
   static SUCCESS_MSG_COLOR = 'green';
   static FAILURE_MSG_COLOR = 'red';
 }
+
+export class ScaffoldTreeFilter extends DG.Filter {
+  viewer: ScaffoldTreeViewer = new ScaffoldTreeViewer();
+  savedTree: string = '';
+  
+  constructor() {
+    super();
+    this.root = ui.divV([]);
+    this.subs = [];
+  };
+
+  get isFiltering(): boolean {
+    return super.isFiltering;
```

Comment:
isFiltering indicates whether the filter is filtering something (in this case, whether a user clicked on a substructure to filter on), or is it just in its default state. And anyway, why would you return super.isFiltering?

Reply:
isFiltering is needed here in order to check whether the filter has been disabled by the user. 
According to docs:  * This is used to minimize the number of unnecessary computations.
     * Make sure to call super.isFiltering to check whether the filter has been disabled by the user *
It is really useful, especially when scaffold tree viewer performs filtering itself, this is the way to control it. Same way our substructure filter works and others.